integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java

```diff
@@ -289,10 +290,11 @@ public void testOperatorRestartWhenPodRoll() {
         replicaCount,
         adminSecretName);
 
-    // wait till rolling restart has started by checking admin server pod is deleted
-    logger.info("Waiting for rolling restart to start by checking {0} pod is deleted in namespace {0}",
+    // wait till rolling restart has started by checking admin server pod has restarted
```

Comment:
The test case, as its name indicated, is intended to test operator restart DURING the managed servers are rolling-restarted. This change made it testing operator restart AFTER the managed servers' rolling-restart completes.

Reply:
For the rolling restart, first admin pod is restarted and then managed server pods, so here the test waits for the admin pod to be restarted and then restarts operator while the managed servers are getting rolled.

Reply:
I think that makes sense. thanks.